packages/wallet-sdk/src/sign/extension/ExtensionSigner.ts

```diff
@@ -0,0 +1,63 @@
+import { StateUpdateListener } from '../interface';
+import { AppMetadata, RequestArguments, Signer } from ':core/provider/interface';
+import { AddressString } from ':core/type';
+import { CBInjectedProvider, CBWindow } from ':util/provider';
```

Comment:
why do we need this `CBInjectedProvider`

Reply:
good catch. [7597427](https://github.com/coinbase/coinbase-wallet-sdk/pull/1332/commits/75974278f9e040125148b89172520aa5bc43ac70)

Reply:
actually we need it on the injected provider to make TS happy about setAppInfo in the constructor. Updated: [ef62054](https://github.com/coinbase/coinbase-wallet-sdk/pull/1332/commits/ef620542631a2df0ed4b812d3fc8a360b5502cfd)